docs/_static/worker_start.sh

```diff
@@ -2,7 +2,7 @@
  
 # A simple script used by Red Hat to start teuthology-worker processes.
  
-ARCHIVE=$HOME/archive
+ARCHIVE=${ARCHIVE:-"$HOME/archive"}
```

Comment:
could also put

```sh
: ${ARCHIVE:="$HOME/archive"}
```

see https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Just curious, why do you think it is better than ":-"?

Reply:
less repeating.